patch_account_list.sh

```
@@ -0,0 +1,22 @@
+cat << 'INNER_EOF' > /tmp/account_list.patch
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)